sdk/messaging/azservicebus/message.go

```diff
@@ -347,3 +373,16 @@ func uuidFromLockTokenBytes(bytes []byte) (*amqp.UUID, error) {
 
 	return &amqpUUID, nil
 }
+
+func asInt64(v interface{}, defVal int64) int64 {
```

Comment:
When would `defVal` be returned and is this an expected scenario or just defensive programming?  Also, if it would always be zero, consider removing the arg and returning zero in the `default` clause.

Reply:
It's defensive at the moment: if the value wasn't there (ie, was 'nil' or incorrectly typed). 

For today, zero would be right but it's also nice to have that documented locally (ie, at the call site) since it's pretty specific to the usage (ie, 'nil' might just mean the property was optional).